guides/paper-iron-layout.md

```diff
@@ -0,0 +1,320 @@
+        href="/bower_components/iron-flex-layout/iron-flex-layout.html">
+...
+<body class="fullbleed vertical layout">
+  <paper-header-panel class="flex">
+    <paper-toolbar>
+      <div>Header</div>
+      <span class="flex"></span>
+      <paper-icon-button icon="search"></paper-button-icon>
+    </paper-toolbar>
+    <div>Content</div>
+  </paper-header-panel>
+</body>
+```
+
+[Demonstration](assets/icons.html)
+
+`paper-icon-button` displays the icon and handles the icon's behavior.
+`iron-icons` is a collection of SVG icons which you can use for free 
+in your project. 
+
+How does the search icon display on the right side? The trick
+is the `span` between the `div` and the `paper-icon-button`. 
+The `div` containing the text `Header` only takes up as 
+much space as is needed to display 
+the text content. Same with the `paper-icon-button`; it only takes up
+as much space as is needed to display the icon. The `flex`
+class forces the `span` to fill the entire space between the `div` and
+the `paper-icon-button`.
+
+### Setting the height
+
+Use the `medium-tall` (2x regular height) and `tall` (3x regular height) style 
+classes to change the height of your header.
+
+```hmtl
+...
+<head>
+...
+  <link rel="import" 
+        href="/bower_components/paper-header-panel/paper-header-panel.html">
+  <link rel="import" 
+        href="/bower_components/paper-toolbar/paper-toolbar.html">
+  <link rel="import" 
+        href="/bower_components/iron-flex-layout/iron-flex-layout.html">
+...
+<body class="fullbleed vertical layout">
+  <paper-header-panel class="flex">
+    <paper-toolbar class="tall">
+      <div>Header</div>
+    </paper-toolbar>
+    <div>Content</div>
+  </paper-header-panel>
+</body>
+...
+```
+
+[Demonstration](assets/tall-header.html)
+
+### Adding tabs
+
+Use `paper-tabs` to add tabs to your header:
+
+```hmtl
+...
+<head>
+...
+  <link rel="import" 
+        href="/bower_components/paper-header-panel/paper-header-panel.html">
+  <link rel="import" 
+        href="/bower_components/paper-toolbar/paper-toolbar.html">
+  <link rel="import" 
+        href="/bower_components/paper-icon-button/paper-icon-button.html">
+  <link rel="import"
+        href="/bower_components/paper-tabs/paper-tabs.html">
+  <link rel="import" 
+        href="/bower_components/iron-icons/iron-icons.html">
+  <link rel="import" 
+        href="/bower_components/iron-flex-layout/iron-flex-layout.html">
+...
+<body class="fullbleed vertical layout">
+  <paper-header-panel class="flex">
+    <paper-toolbar class="medium-tall">
+      <paper-icon-button id="navicon"
+                         icon="menu"></paper-icon-button>
+      <!-- flex class forces span to fill space between icons -->
+      <span class="flex">Title</span>
+      <!-- icon displays at right because of span class above -->
+      <paper-icon-button id="morebutton"
+                         icon="more-vert"></paper-icon-button>
+      <paper-tabs class="bottom fit" selected="0">
```

Comment:
I tried removing it. The demo appears to break. So I'm going to leave it in. 

With `fit`:

![image](https://cloud.githubusercontent.com/assets/4713486/9918926/9913a154-5c7e-11e5-87e5-f22dd63ada6e.png)


Without `fit`:

![image](https://cloud.githubusercontent.com/assets/4713486/9918907/84f4164a-5c7e-11e5-994e-515d671a4508.png)